<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoPop</class>
 <widget class="QDialog" name="UserInfoPop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1,1">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>用户ID *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_id"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>用户名 *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="le_name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>所属部门</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cm_dept"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,3">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>用户权限</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cm_kind"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1,1,1,1">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="save_btn">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancle_btn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
